### DISCUSSION TITLE
Feature: Ability to right-click objects behind other objects when the front object is see-through

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenRCT2/OpenRCT2#15897] [Feature requests]
Idea: When rides, scenery etc. are "see-through", you should be able to right-click objects behind them without actually clicking the see-through object. This would allow path manipulation or scenery removal without having to first remove/modify what is in the foreground first. It would save a lot of hassle.

I think something similar has already been done in OpenLoco, which essentially does exactly what I am suggesting here.

**Comment by MASHzj** [comment]
This is possible in two ways (both accessible via View options menu):
- Transparency options : by clicking on the eye-icons you can set objects to invisible instead of see-through
- Cut-away view: when an object is blocked by another object of the same type, this is an alternative. You can do this by selecting the area where the object you want to access is on.

**Comment by J-PIE-314** [comment]
This feature was implemented at some point in development.
